packages/ui-components/package.json

```diff
@@ -15,8 +15,10 @@
     "test": "echo skipped"
   },
   "dependencies": {
+    "@polkadot/ui-app": "^0.29.1",
     "@polkadot/ui-identicon": "^0.36.1",
     "@polkadot/ui-keyring": "^0.36.1",
+    "@polkadot/ui-signer": "^0.29.1",
```

Comment:
@jacogr you might be interested: while pulling ui-app, I somehow also needed ui-signer

Reply:
Ummm.... Sorry :( Dunno why, will take a look.